packages/styles/settings-tools/_s.inline-icons.scss

```diff
@@ -26,4 +26,55 @@
   @if $icon == 'delete-24' {
     @return '<svg xmlns="http://www.w3.org/2000/svg" fill="#{$fill}" viewBox="0 0 24 24"><path d="M18.08 8a1 1 0 0 0-1.08.92L16.08 20H7.92L7 8.92a1 1 0 1 0-2 .16l1 12A1 1 0 0 0 7 22h10a1 1 0 0 0 1-.92l1-12A1 1 0 0 0 18.08 8z"/><path d="M19 5h-3.77l-.65-2.27a1 1 0 0 0-1-.73h-3.2a1 1 0 0 0-1 .73L8.77 5H5a1 1 0 0 0 0 2h14a1 1 0 0 0 0-2zm-8.62-2h3.24l.57 2H9.81zM12.5 18v-8a.5.5 0 0 0-1 0v8a.5.5 0 0 0 1 0zM14.25 18.5a.5.5 0 0 0 .5-.47l.5-8a.5.5 0 0 0-.47-.53.49.49 0 0 0-.53.47l-.5 8a.5.5 0 0 0 .47.53zM9.75 18.5a.5.5 0 0 0 .47-.53l-.5-8a.49.49 0 0 0-.53-.47.5.5 0 0 0-.47.53l.5 8a.5.5 0 0 0 .53.47z"/></svg>';
   }
+
+  // Empty
+  @if $icon == 'star-empty-16' {
+    @return '<svg xmlns="http://www.w3.org/2000/svg" fill="#{$fill}" viewBox="0 0 16 16"><path d="M8 11.95L4.29 14 5 9.66 2 6.58l4.15-.63L8 2l1.85 3.95 4.15.63-3 3.08.71 4.34L8 11.95z"/><path d="M8 11.95L4.29 14 5 9.66 2 6.58l4.15-.63L8 2l1.85 3.95 4.15.63-3 3.08.71 4.34L8 11.95z" data-name="path-1"/></svg>';
```

Comment:
duplicated path, with a data-name attribute in excess

Reply:
Fixed